### PR TITLE
ui: improvement in workflow tab

### PR DESCRIPTION
**Pull request by masco-enovance**
added a loading icon till workflow return a result

### CODE REVIEW

File: statics/js/components/workflow.js [hunk 1]
```diff
@@ -88,6 +88,8 @@ Vue.component('workflow-params', {
   methods: {
 
     submit: function() {
+      this.display.result = true;
+      this.result.value = undefined;
       var self = this;
```
Comment (sergeyglazyrindev-samsung) [inline statics/js/components/workflow.js]: Hello
Could you please wrap these two lines into one function and call it like toggleReadyStatus ?
It's better to turn on/off flags in setters,

File: statics/js/components/workflow.js [hunk 2]
```diff
@@ -153,19 +159,22 @@ Vue.component('workflow-call', {
...
+          <i class="fa fa-circle-o-notch fa-spin fa-2x fa-fw"></i>
+        </div>
       </div>
     </div>
   `,
 
   watch: {
     'currentWorkflow': function () {
-      this.result.value = undefined
+      this.display.result = false;
```
Comment (sergeyglazyrindev-samsung) [inline statics/js/components/workflow.js]: Move it to the function: toggleReadyStatus

Comment (masco-enovance) [reply statics/js/components/workflow.js]: done

File: statics/js/components/workflow.js (hunk 1 above)
Comment (masco-enovance) [reply statics/js/components/workflow.js]: done

**Comment by masco-enovance** [comment]
run skydive-k8s-tests
run skydive-functional-tests-backend-orientdb
run skydive-functional-tests-backend-elasticsearch

**Comment by masco-enovance** [comment]
run skydive-k8s-tests

**Comment by masco-enovance** [comment]
run skydive-k8s-tests

**Comment by masco-enovance** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by masco-enovance** [comment]
run skydive-k8s-tests

**Pull request merged**